test/functionalTest/cases/1187_empty_attribute_in_create/empty_attribute_in_create.test

```diff
@@ -32,8 +32,8 @@ brokerStart CB
 #
 # 01. POST /v2/entities to create E1/A1=""
 # 02. GET /v2/entities/E1 to see A1==""
-# 03. POST /v2/entities to create E1/A2="", using json object
-# 04. 04. GET /v2/entities/E1 to see A2==""
+# 03. POST /v2/entities to create E1/A2="", using json object (fails, as entity already exists)
```

Comment:
Except there is no POST on /v2/entities/E1.
There is a second POST on /v2/entities (with E1 in payload).

I understand a second POST /v2/entities with the same *id* in the payload should give an error ...